BUILDING.md

```diff
@@ -28,7 +28,7 @@ On FreeBSD and OpenBSD, you may also need:
 * libexecinfo (FreeBSD and OpenBSD only)
 
 
-```text
+```sh
```

Comment:
This probably should be `console`, but I need to re-check the code highlighting on that.